modules/home/assets/js/components/top-section.js

```diff
@@ -1,7 +1,44 @@
-const TopSection = () => {
+import { Box, Paper } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import StyledYoutubeIcon from '../icons/youtube-icon';
+
+const TopSection = ( { ctaUrl, videoUrl, embedUrl } ) => {
 	return (
-		<p>Add the TopSection component here</p>
+		<Paper justifyContent="space-between" alignItems="flex-start" elevation={ 0 } sx={ { display: 'flex', flexDirection: 'row', bgcolor: 'theme.palette.background.default', mt: 6 } }>
+			<Box ml={ 4 } my={ 4 }>
+				<Box sx={ { display: 'flex', flexDirection: 'column', width: '85%', mb: 3 } }>
+					<Typography variant="body2">Hi!</Typography>
+					<Typography variant="h6">Welcome to Elementor</Typography>
+					<Typography variant="body2" >Get introduced to Elementor by watching our &ldquo;Getting Started&rdquo; video series. It will guide you through the steps needed to create your website. Then click to create your first page.</Typography>
+				</Box>
+				<Box>
+					<Button variant="contained" size="small" sx={ { mr: 1, mt: { xs: 0, sm: 1, md: 0, lg: 0, xl: 0 } } } href={ ctaUrl } target="_blank">
+						<Typography variant="button">Create a Page</Typography>
+					</Button>
+					<Button variant="outlined" color="secondary" size="small" startIcon={ <StyledYoutubeIcon /> } href={ videoUrl } target="_blank" sx={ { mt: { xs: 1, sm: 1, md: 0, lg: 0, xl: 0 } } }>
+						<Typography variant="button">Watch a quick starter</Typography>
+					</Button>
+				</Box>
+			</Box>
+			<Box my={ 4 } mr={ 4 }>
+				<iframe
```

Comment:
@rotemee Do we have a standard solution to embed Youtube Iframes?

Reply:
done